modules/twinkleconfig.js

```diff
@@ -1689,12 +1689,14 @@ Twinkle.config.writePrefs = function twinkleconfigWritePrefs(pageobj) {
 		"// changing the configuration parameters in a valid-JavaScript way) will be\n" +
 		"// overwritten the next time you click \"save\" in the Twinkle preferences\n" +
 		"// panel.  If modifying this file, make sure to use correct JavaScript.\n" +
+	    	"// <no" + "wiki>\n"+
```

Comment:
I always get confused by `<nowiki>` functionality, but doesn't the fact that the page is wrapped with `<nowiki>..</nowiki>` mean that the splitting is unnecessary?  Or does it break anyway?

Reply:
It will break anyway, since, without splitting the tags, the gadget file would be parsed as <nowiki>AAA<nowiki>BBB</nowiki>CCC</nowiki>, which doesn't work as expected - a nowiki tag within another nowiki tag is ignore - they can't be nested. See https://en.wikipedia.org/wiki/User:DannyS712/sandbox#Nowiki_test for a demonstration

Reply:
Sure, in wikitext, but on a js contentModel page?  See https://en.wikipedia.org/wiki/User:Amorymeltzer/nowiki.js

Reply:
Yes, the problem is _after_ the first nowiki tag is closed (sorry, in my previous edit the brackets didn't display, I've fixed that) the remaining content is no longer escaped - see https://en.wikipedia.org/wiki/User:DannyS712/sandbox.js.

Reply:
Right you are!  No wonder I get tripped up.  Totally lame!